docs/using/recipe_code/custom_environment.dockerfile

```diff
@@ -28,12 +28,37 @@ RUN "${CONDA_DIR}/envs/${env_name}/bin/python" -m ipykernel install --user --nam
 RUN "${CONDA_DIR}/envs/${env_name}/bin/pip" install --no-cache-dir \
     'flake8'
 
-# Uncomment this section to activate custom environment by default
+# This changes the custom Python kernel so that the custom environment will
+# be activated for the respective Jupyter Notebook and Jupyter Console
+# hadolint ignore=DL3059
+RUN python <<HEREDOC
+import json
+import os
+from pathlib import Path
+
+env_name = "${env_name}"
+CONDA_DIR = os.environ["CONDA_DIR"]
```

Comment:
Removed CONDA_DIR hardcore here, we usually use `${CONDA_DIR}` value in our docker files.

Reply:
Thoughts on turning this into a file that can be called, rather than HEREDOC? Especially given python is indent sentisitive, I think HEREDOC can be difficult to modify

Reply:
I thought about it, but then I thought - but we would like to make our recipe easily-copyable (and then it should run).
But the I thought - let's add it as `/opt/setup-scripts/activate_notebook_custom_env.py`
People, who don't use it, won't notice, but people using custom env will still be happy (and their Dockerfiles) will be shorter - so thank you for suggestion, done.